packages/clay-charts/src/index.tsx

```diff
@@ -1,31 +1,32 @@
 /**
- * © 2018 Liferay, Inc. <https://liferay.com>
+ * © 2019 Liferay, Inc. <https://liferay.com>
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
+
+import BaseChart, {IProps, Types} from './BaseChart';
+import Billboard from './BillboardWrapper';
 import React from 'react';
-import * as ReactBillboard from 'react-billboardjs';
-import ClayChart from './Chart';
 
-const setType = type => ({data, ...otherProps}) => (
-	<ClayChart {...otherProps} data={{...data, type}} />
+const setType = (type: Types) => ({data, ...otherProps}: IProps) => (
+	<BaseChart {...otherProps} data={{...data, type}} />
 );
 
-export const AreaLineChart = setType('area-line');
+export const AreaLineChart = setType('area-line-range');
```

Comment:
Usually I think we namespace with the `Component.***` for sub-components. Such as `Card.Header`. These charts are technically the high-level components, so it may make sense to change them to `ClayChartBar`, or we remove these specific named components and just opt that people always pass in the type to `ClayChart`.

Reply:
Hm yeah, makes sense. I think it's worth delivering just the `ClayChart` and removing the other components.

Reply:
Yeah I agree, I think before added this to keep the API the same as clay-charts in soy.